index.html

```diff
@@ -164,6 +164,39 @@
     </nav>
 
     <main>
+      <div class="roles-container">
+        <div class="role-details">
+          <h2 class="role-heading">Select your role</h2>
+          <div class="role-details-field">
+            <label class="checkbox-label">
+              <input type="checkbox" name="developer" class="checkbox-input" />
+              Developer
```

Comment:
will you please move copy to constant files

Reply:
Didn't understand this comment.
It is html file

Reply:
@shubhamsigdar1 why you resolved this conversation? You move all constants in separate constants file?

Reply:
Done with changes. can you please check latest commit message